lib/modules/blockchain_process/dev_funds.js

```diff
@@ -34,37 +34,44 @@ class DevFunds {
   }
 
   _sendTx() {
-    if (this.networkId !== 1337) {
+    // Send TXs only in dev networks
+    if (this.networkId !== 1337 && this.networkId !== 17) {
       return;
     }
     this.web3.eth.sendTransaction({value: "1000000000000000", to: "0xA2817254cb8E7b6269D1689c3E0eBadbB78889d1", from: this.web3.eth.defaultAccount});
   }
 
-  // trigger regular txs due to a bug in geth and stuck transactions in --dev mode
   _regularTxs(cb) {
     const self = this;
     self.web3.eth.net.getId().then((networkId) => {
       self.networkId = networkId;
       if (self.networkId !== 1337) {
         return;
       }
-
-      setInterval(function () { self._sendTx(); }, 1500);
+      setInterval(function() { self._sendTx(); }, 1500);
       if (cb) {
         cb();
       }
     });
   }
 
-  _regularUnlocks() {
-    const self = this;
-    setInterval(function () { self.unlockAccounts(self.password, () => { }); }, 20000);
+  _fundAccounts(balance, cb) {
+    async.each(this.accounts, (account, next) => {
+      this.web3.eth.getBalance(account).then(currBalance => {
+        const remainingBalance = balance - currBalance;
+        if (remainingBalance <= 0) return next();
+        this.web3.eth.sendTransaction({to: account, value: remainingBalance}).catch(next);
```

Comment:
It was on this catch that I specified putting a console.error. Sorry for the misunderstanding.

Reply:
Done.